Documentation/infra-guide/arm64-infra.md

```diff
@@ -55,6 +55,14 @@ When a member is removed from the infra admins table existing members must revie
 
 Note: When revoking access do not delete a user or their home directory from servers, as access may need to be reinstated in future.
 
-## Regular access review
+### Regular access review
 
 On a regular at least quarterly basis members of the infra admins team are responsible for verifying that no unneccessary infrastructure access exists by reviewing membership of the table above and existing server access.
+
+## Provisioning new machines
```

Comment:
Could we get any list of things we expect a fresh machine will need? Aka github action agent, docker, etc.

Reply:
Certainly - that will be my next task after this pr, under https://github.com/etcd-io/etcd/issues/15954

Reply:
SG

Reply:
Copy paste from past conversation: 

> Hey Dims, do you know how the etcd self hosted github runner environment is set up?

> hi! i used the github docs around self-hosted github runners - https://docs.github.com/en/actions/hosting-your-own-runners/managing-self-hosted-runners/about-self-hosted-runners

> here's the steps to add a runner to a repo i used https://docs.github.com/en/actions/hosting-your-own-runners/managing-self-hosted-runners/adding-self-hosted-runners#adding-a-self-hosted-runner-to-a-repository